api/capi/src/nnstreamer-capi-single.c

```diff
@@ -86,8 +86,7 @@ typedef enum
 {
   IDLE = 0,           /**< ready to accept next input */
   RUNNING,            /**< running an input, cannot accept more input */
-  JOIN_REQUESTED,     /**< should join the thread, will exit soon */
-  ERROR               /**< error on thread, will exit soon */
+  JOIN_REQUESTED      /**< should join the thread, will exit soon */
```

Comment:
If you remove an element from this public-API-header, you will be required to process ACR. Do you really think this removal is required? (you may keep it as a "reserved" value.)